src/index.njk

```diff
@@ -10,7 +10,7 @@ layout: layout.njk
       {% for page in archivePages %}
       <li class="border-t first:border-t-0">
         <a
-          href="{{ page.url }}"
+          href="{{ page.pathname }}"
```

Comment:
Is this because we're using relative urls here, right?

Reply:
Yes